wit-manifest.json

```diff
@@ -248,5 +248,10 @@
         "commit": "478f67af71b126e47cfb0839c6238c07f7e2bb2b",
         "name": "example-buserror",
         "source": "git@github.com:sifive/example-buserror.git"
+    },
+    {
+        "commit": "b14ba0a76259b5ffa60feb1b0249e7753bcbcf01",
+        "name": "example-hardware-performance-monitor",
+        "source": "git@github.com:sifive/example-hardware-performance-monitor.git"
```

Comment:
In the example, instead of /* Read cycle count */, it should describe the purpose, like /* Number of cycle taken before Retire Instruction or Cache-Missed events occurred */

Reply:
fixed.